source/shared/cpp/ObjectModel/SharedAdaptiveCard.cpp

```diff
@@ -231,10 +236,11 @@ std::shared_ptr<AdaptiveCard> AdaptiveCard::MakeFallbackTextCard(
 #else
```

Comment:
Won't this break Android? Doesn't it also need the speak parameter?

Reply:
I think you still need to run swig for this change.

---
In reply to: [216856776](https://github.com/Microsoft/AdaptiveCards/pull/1889#discussion_r216856776) [](ancestors = 216856776)